triton-server/src/download/schema.sql

```diff
@@ -75,5 +75,6 @@ create table constants (
     id                  INTEGER PRIMARY KEY ASC check(id = 1),
     http_project_size   INTEGER not null,
     globus_project_size INTEGER not null,
-    sftp_project_size   INTEGER not null
+    sftp_project_size   INTEGER not null,
+    extension_size      INTEGER not null default 7
```

Comment:
expiry_days ? expiration_delay ? extension_size sends me images of hair ...

Reply:
![image](https://github.com/c3g/triton/assets/15523177/54303e27-0714-4b3f-bd0b-34e14bd4c2f1)